Content/Items/Manabonds/AquaticManabond.cs

```diff
@@ -0,0 +1,184 @@
+﻿using StarlightRiver.Helpers;
+using System.Collections.Generic;
+using Terraria.Graphics.Effects;
+using Terraria.ID;
+
+namespace StarlightRiver.Content.Items.Manabonds
+{
+	internal class AquaticManabond : Manabond
+	{
+		public override string Texture => AssetDirectory.ManabondItem + Name;
+
+		public AquaticManabond() : base("Aquatic Manabond", "Your minions can store 40 mana\nYour minions siphon 6 mana per second from you untill full\nYour minions spend 8 mana to attack with a bouncing waterbolt occasionally") { }
+
+		public override void SafeSetDefaults()
+		{
+			Item.value = Item.sellPrice(gold: 15);
+			Item.rare = ItemRarityID.Orange;
+		}
+
+		public override void MinionAI(Projectile minion, ManabondProjectile mp)
+		{
+			if (mp.timer % 50 == 0 && mp.mana >= 8 && mp.target != null)
+			{
+				mp.mana -= 8;
+				Projectile.NewProjectile(minion.GetSource_FromThis(), minion.Center, Vector2.Normalize(mp.target.Center - minion.Center) * 6, ModContent.ProjectileType<Waterbolt>(), 24, 1f, minion.owner);
+			}
+		}
+
+		public override void AddRecipes()
+		{
+			Recipe recipe = CreateRecipe();
+			recipe.AddIngredient(ModContent.ItemType<BasicManabond>(), 1);
+			recipe.AddIngredient(ItemID.WaterBolt, 1);
+			recipe.AddTile(TileID.Bookcases);
+			recipe.Register();
+		}
+	}
+
```

Comment:
Some specificity in the naming here could be nice, its niche but there are 2 vanilla things also called Waterbolt. Something like AquaticWaterbolt, or ManabondWaterbolt should be used instead imo

Reply:
Not sure this is a huge concern as its virtually impossible to conflate a modded entity with a vanilla one due to differences in how their ID is refferenced

Reply:
its less of a concern programmatically but more a concern of consistency with other items. Other projectiles tied to items are usually named as such, as apposed to something as nondescript as "Waterbolt"